conda/libmesh/meta.yaml

```diff
@@ -1,5 +1,5 @@
 # Do not use jinja templating (A physical change to this file is required to trigger a build)
-{% set build = 0 %}
+{% set build = 1 %}
```

Comment:
I know we're upping the build for consistency but this shouldn't actually change the libmesh build at all, yeah?

Reply:
It should not. This is just to bring the manual build in line with the conda package. I waffled on upping the build number, but since I am modifying `build.sh` I thought I should stick to that rule we've set for ourselves.

Reply:
Yeah, I'm fine with that.